README.md

```diff
@@ -13,7 +13,7 @@ Before building the updater itself you need to build aria2 library
 ```
 cd aria2
 autoreconf -i
-./configure --without-libxml2 --without-libexpat --without-sqlite3 --enable-libaria2 --without-zlib --without-libcares --enable-static=yes ARIA2_STATIC=yes --without-libssh2 --disable-websocket --disable-nls
+./configure --without-libxml2 --without-libexpat --without-sqlite3 --enable-libaria2 --without-zlib --without-libcares --enable-static=yes ARIA2_STATIC=yes --without-libssh2 --disable-websocket --disable-nls --with-openssl
```

Comment:
What's our strategy for this? Are we statically compiling OpenSSL or dlopen'ing it?

Reply:
It's static. You can check the commit message - I diffed the outputs of `ldd` to make sure we aren't adding unwanted dynamic dependencies.

Reply:
aight. FYI, 

`objdump -x <file> | grep NEEDED` is probably nicer than ldd. ldd includes transitive dependencies whereas objdump -x only shows the directly needed deps.

Reply:
Interesting, I tried this and it showed one additional added dependency `libXext.so.6`.